spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+describe ApplicationHelper do
+  describe '#page_title' do
+    context 'given a page that does not exist' do
+      it 'returns the application title' do
+        page_title = helper.page_title(page: :meow)
```

Comment:
do we need a hash params for the page, it is just a one param, what about
`page_title_for(:meow)`

Reply:
Makes sense!